exporters/trace/fluentforward/fluentforward.go

```diff
@@ -0,0 +1,199 @@
+package fluentforward
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net"
+	"time"
+
+	apitrace "go.opentelemetry.io/otel/api/trace"
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/label"
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+type ffSpan struct{
+	_msgpack struct{}            `msgpack:",asArray"`
+	Tag string                   `msgpack:"tag"`
+	Ts int64                     `msgpack:"ts"`
+	SpanData spanData						 `msgpack:"spanData"`
+}
+
+type spanData struct{
+	SpanContext spanContext         `msgpack:"spanContext"`
+	ParentSpanId string             `msgpack:"parentSpanId"`
+	SpanKind apitrace.SpanKind      `msgpack:"spanKind"`
+	Name string                     `msgpack:"name"`
+	StartTime int64                 `msgpack:"startTime"`
+	StatusMessage string            `msgpack:"statusMessage"`
+	EndTime int64                   `msgpack:"endTime"`
+	Attrs map[label.Key]interface{} `msgpack:"attrs"`
+	DroppedAttributeCount int       `msgpack:"droppedAttributesCount"`
+	DroppedMessageEventCount int    `msgpack:"droppedMessageEventCount"`
+	DroppedLinkCount int            `msgpack:"droppedLinkCount"`
+	StatusCode string               `msgpack:"statusCode"`
+	Links []link                    `msgpack:"links"`
+}
+
+type spanContext struct{
+	TraceId string  `msgpack:"traceId"`
+	SpanId string   `msgpack:"spanId"`
+	TraceFlags byte `msgpack:"TraceFlags"`
```

Comment:
```suggestion
	TraceFlags byte `msgpack:"traceFlags"`
```

?

Reply:
what's this?

Reply:
[According to the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#spancontext), In the current version, I believe that it only supports one flag, that tells you if the traces have been sampled.